robot.yaml

```diff
@@ -1,12 +1,24 @@
+# For more details on the format and content:
+# https://github.com/robocorp/rcc/blob/master/docs/recipes.md#what-is-in-robotyaml
+
 tasks:
   Run all tasks:
```

Comment:
I would rename this to something more meaningful as this is the task name shown in the Control Room. At least comment recommending renaming this to match the actual task.